Source/WebGPU/WebGPU/Adapter.mm

```diff
@@ -150,49 +150,49 @@
 
 void wgpuAdapterReference(WGPUAdapter adapter)
 {
-    WebGPU::fromAPI(adapter).ref();
+    WebGPU::protectedFromAPI(adapter)->ref();
```

Comment:
This seems unnecessary. We're literally calling ref() which is an inline trivial function.

Reply:
Agreed, will fix up the *Reference / *Release calls

Reply:
(resolving as we discussed offline)